Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.16-alpine3.13 AS builder
+FROM golang:1.16-alpine3.15 AS builder
```

Comment:
I recently [added support](https://github.com/golang-migrate/migrate/commit/30411f49a1855cea079f7c736a430bd36ca8f654) for Go 1.19 so could you change this to Go 1.19 and Alpine Linux 3.16 and update the other images accordingly?